stable-diffusion/stable-diffusion-xl-1.0-trt/config.yaml

```diff
@@ -15,6 +15,8 @@ model_metadata:
 model_name: Stable Diffusion XL - TensorRT
 model_framework: custom
 model_type: custom
+model_cache:
+- repo_id: "baseten/sdxl-1.0-trt-8.6.1.post1-engine"
```

Comment:
Can we also add https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0 and https://huggingface.co/stabilityai/stable-diffusion-xl-refiner-1.0?